app/create_app.py

```diff
@@ -1,14 +1,26 @@
+import os
+
 from flask import Flask
 from flask_compress import Compress
+from flask_seasurf import SeaSurf
 from flask_talisman import Talisman
-from jinja2 import ChoiceLoader, PackageLoader, PrefixLoader
+from jinja2 import ChoiceLoader
+from jinja2 import PackageLoader
+from jinja2 import PrefixLoader
+
 
 app = Flask(__name__, static_url_path="/assets")
+app.config["SECRET_KEY"] = os.environ.get("CSRF_KEY")
```

Comment:
why is this being set here and not in the config file?

Reply:
Moved it to config.py file